src/amo/components/RatingManager/styles.scss

```diff
@@ -12,6 +12,9 @@
   padding: 2px 16px;
   position: absolute;
   width: 100%;
+  top:50%;
+  left: 50%;
+  transform: translate(-50%,-50%)
```

Comment:
Hi @HosseinMarzban can we convert this whole section to something like this as i am suggesting here. I think that will solve the `stylelint` and `prettier` issues.
```
.RatingManager-log-in-to-rate-button {
  font-size: $font-size-s;
  padding: 2px 16px;
  position: absolute;
  left: 50%;
  top: 50%;
  transform: translate(-50%, -50%);
  width: 100%;
}
```

Reply:
This is exactly what we need!